qa-visualiser/README.md

```diff
@@ -0,0 +1,48 @@
+# QA visualiser
+
+Python project for creating visualisations of the different question pathways for each topic, to support QA testing.
+
+## Overview
+
+The tool makes a call to a cms endpoint within the plan-tech web app that returns all sections from Contentful.
+Then for each section, it generates a graph of all possible pathways through the section by making every question a node, and every link between an answer and its next question an edge.
+
+The `models.py` file uses Pydantic to ensure the data coming back from the cms endpoint has the fields required for the visualisation generator to work.
+The models do not contain all the fields that will be present within the json content but just those that are essential for the generator.
+Extra fields are allowed but missing fields raise a validation error.
+
+## Local setup
+
+To run the qa-visualiser locally:
+1. Install [uv](https://github.com/astral-sh/uv)
+2. Setup your .env file by copying .env.example and filling in the key-value pairs
```

Comment:
Is it worth adding some detail on what the environment variables are? Someone might get confused about the API key and use the Contentful API key. I.e. I did that.

Reply:
good shout ^